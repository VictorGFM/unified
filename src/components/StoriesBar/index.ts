import StoriesBarDiv from './StoriesBar'

export default StoriesBarDiv
