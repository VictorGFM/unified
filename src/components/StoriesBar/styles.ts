import styled from 'styled-components'
import { COLORS } from '../../styles/global'

export const StoriesBarDiv = styled.div`
  display: flex;
  position: relative;
  align-items: center;
  justify-content: center;

  width: 600px;
  height: 80px;
  border: solid ${COLORS.GREY.THREE} 1px;
  margin: 15px 0px;

  background-color: ${COLORS.WHITE.ONE};
  border-radius: 7px;
  box-sizing: border-box;
`
export const ContactStoriesDiv = styled.div`
  display: flex;
  justify-content: space-evenly;

  width: 540px;
`
