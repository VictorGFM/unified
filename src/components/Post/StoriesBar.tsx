import { PostDiv } from './styles'

const Post = () => {
  return <PostDiv>TEST</PostDiv>
}

export default Post
