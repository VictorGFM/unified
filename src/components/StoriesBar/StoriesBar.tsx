import { StoriesBarDiv } from './styles'

const StoriesBar = () => {
  return <StoriesBarDiv>TEST</StoriesBarDiv>
}

export default StoriesBar
