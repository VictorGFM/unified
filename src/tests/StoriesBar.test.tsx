import { describe, expect, it } from 'vitest'
import { render } from '@testing-library/react'
import StoriesBar from '../components/StoriesBar'

describe('<StoriesBar/>', () => {
  it('should show match snapshot', () => {
    const component = render(<StoriesBar />)

    expect(component).toMatchSnapshot()
  })
})
