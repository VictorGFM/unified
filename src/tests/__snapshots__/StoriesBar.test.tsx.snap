// Vitest Snapshot v1

exports[`<StoriesBar/> > should show match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-gKXOVf eMtIrj"
      >
        <div
          class="sc-eCYdqJ kNmmNQ"
          direction="left"
        >
          <img
            class="sc-jSMfEi gxwlYz"
            src="/src/assets/icons/ArrowIcon.svg"
          />
        </div>
        <div
          class="sc-iBkjds hCHhzn"
        >
          <div
            class="sc-bczRLJ mKoYI"
          >
            <div
              class="sc-gsnTZi liUpCO"
            >
              <img
                class="sc-dkzDqf kmAHmU"
                src="/src/assets/icons/AlfredoPhotoIcon.svg"
              />
            </div>
            <span
              class="sc-hKMtZM fqAUFi"
            >
              Alfredo
            </span>
          </div>
          <div
            class="sc-bczRLJ mKoYI"
          >
            <div
              class="sc-gsnTZi liUpCO"
            >
              <img
                class="sc-dkzDqf kmAHmU"
                src="/src/assets/icons/ContactPhotoIcon.svg"
              />
            </div>
            <span
              class="sc-hKMtZM fqAUFi"
            >
              Contact Name
            </span>
          </div>
          <div
            class="sc-bczRLJ mKoYI"
          >
            <div
              class="sc-gsnTZi liUpCO"
            >
              <img
                class="sc-dkzDqf kmAHmU"
                src="/src/assets/icons/ContactPhotoIcon.svg"
              />
            </div>
            <span
              class="sc-hKMtZM fqAUFi"
            >
              Contact Name
            </span>
          </div>
          <div
            class="sc-bczRLJ mKoYI"
          >
            <div
              class="sc-gsnTZi liUpCO"
            >
              <img
                class="sc-dkzDqf kmAHmU"
                src="/src/assets/icons/ContactPhotoIcon.svg"
              />
            </div>
            <span
              class="sc-hKMtZM fqAUFi"
            >
              Contact Name
            </span>
          </div>
          <div
            class="sc-bczRLJ mKoYI"
          >
            <div
              class="sc-gsnTZi liUpCO"
            >
              <img
                class="sc-dkzDqf kmAHmU"
                src="/src/assets/icons/ContactPhotoIcon.svg"
              />
            </div>
            <span
              class="sc-hKMtZM fqAUFi"
            >
              Contact Name
            </span>
          </div>
          <div
            class="sc-bczRLJ mKoYI"
          >
            <div
              class="sc-gsnTZi liUpCO"
            >
              <img
                class="sc-dkzDqf kmAHmU"
                src="/src/assets/icons/ContactPhotoIcon.svg"
              />
            </div>
            <span
              class="sc-hKMtZM fqAUFi"
            >
              Contact Name
            </span>
          </div>
        </div>
        <div
          class="sc-eCYdqJ blKEqO"
          direction="right"
        >
          <img
            class="sc-jSMfEi gxwlYz"
            src="/src/assets/icons/ArrowIcon.svg"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-gKXOVf eMtIrj"
    >
      <div
        class="sc-eCYdqJ kNmmNQ"
        direction="left"
      >
        <img
          class="sc-jSMfEi gxwlYz"
          src="/src/assets/icons/ArrowIcon.svg"
        />
      </div>
      <div
        class="sc-iBkjds hCHhzn"
      >
        <div
          class="sc-bczRLJ mKoYI"
        >
          <div
            class="sc-gsnTZi liUpCO"
          >
            <img
              class="sc-dkzDqf kmAHmU"
              src="/src/assets/icons/AlfredoPhotoIcon.svg"
            />
          </div>
          <span
            class="sc-hKMtZM fqAUFi"
          >
            Alfredo
          </span>
        </div>
        <div
          class="sc-bczRLJ mKoYI"
        >
          <div
            class="sc-gsnTZi liUpCO"
          >
            <img
              class="sc-dkzDqf kmAHmU"
              src="/src/assets/icons/ContactPhotoIcon.svg"
            />
          </div>
          <span
            class="sc-hKMtZM fqAUFi"
          >
            Contact Name
          </span>
        </div>
        <div
          class="sc-bczRLJ mKoYI"
        >
          <div
            class="sc-gsnTZi liUpCO"
          >
            <img
              class="sc-dkzDqf kmAHmU"
              src="/src/assets/icons/ContactPhotoIcon.svg"
            />
          </div>
          <span
            class="sc-hKMtZM fqAUFi"
          >
            Contact Name
          </span>
        </div>
        <div
          class="sc-bczRLJ mKoYI"
        >
          <div
            class="sc-gsnTZi liUpCO"
          >
            <img
              class="sc-dkzDqf kmAHmU"
              src="/src/assets/icons/ContactPhotoIcon.svg"
            />
          </div>
          <span
            class="sc-hKMtZM fqAUFi"
          >
            Contact Name
          </span>
        </div>
        <div
          class="sc-bczRLJ mKoYI"
        >
          <div
            class="sc-gsnTZi liUpCO"
          >
            <img
              class="sc-dkzDqf kmAHmU"
              src="/src/assets/icons/ContactPhotoIcon.svg"
            />
          </div>
          <span
            class="sc-hKMtZM fqAUFi"
          >
            Contact Name
          </span>
        </div>
        <div
          class="sc-bczRLJ mKoYI"
        >
          <div
            class="sc-gsnTZi liUpCO"
          >
            <img
              class="sc-dkzDqf kmAHmU"
              src="/src/assets/icons/ContactPhotoIcon.svg"
            />
          </div>
          <span
            class="sc-hKMtZM fqAUFi"
          >
            Contact Name
          </span>
        </div>
      </div>
      <div
        class="sc-eCYdqJ blKEqO"
        direction="right"
      >
        <img
          class="sc-jSMfEi gxwlYz"
          src="/src/assets/icons/ArrowIcon.svg"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
